ArchiSteamFarm/SharedInfo.cs

```diff
@@ -49,12 +49,13 @@ internal static class SharedInfo {
 		internal const string StatisticsServer = "asf.justarchi.net";
 		internal const string UlongCompatibilityStringPrefix = "s_";
 		internal const string UpdateDirectory = "_old";
-		internal const string WebsiteDirectory = "www";
 
 		internal static string HomeDirectory => Path.GetDirectoryName(Assembly.GetEntryAssembly().Location);
 		internal static Guid ModuleVersion => Assembly.GetEntryAssembly().ManifestModule.ModuleVersionId;
 		internal static string PublicIdentifier => AssemblyName + (BuildInfo.IsCustomBuild ? "-custom" : "");
 		internal static Version Version => Assembly.GetEntryAssembly().GetName().Version;
+		internal static string CustomWebsiteDirectory => Path.Combine(Directory.GetCurrentDirectory(), "www");
```

Comment:
Now you can check if kestrel requires absolute path here, because `Directory.GetCurrentDirectory()` is almost never needed, you'd just use relative `www`, ASF already switches to given directory when `--path` is given.

Reply:
i know that it switches to the given path ;) that's why i used `Directory.GetCurrentDirectory()` for in the first place. but i'll check

Reply:
yes it needs the full path. using `"./www"` instead of the whole `Path.Combine(...)` call does not work

Reply:
What do you mean it doesn't work? It worked before, so it'd be wise to check why exactly it doesn't now.

Reply:
```internal static string CustomWebsiteDirectory => "./www";``` does give me default ASF-GUI in the browser of my choice despite me having a folder called `www` besides the config-directory.

Reply:
That's better, it means that Kestrel needs that, not `SharedInfo`. I'd suggest keeping shared info as it was before and instead doing `Path.Combine()` magic in kestrel directly, while adding a comment why you're doing `Directory.GetCurrentDirectory()`.

Reply:
And it also means that default `www` directory doesn't need `HomeDirectory` there.